test/helpers/flow_helper_test.rb

```diff
@@ -0,0 +1,110 @@
+require "test_helper"
+
+class FlowHelperTest < ActionView::TestCase
```

Comment:
Should this be accompanied by a deletion of spec/helpers/flow_helper_spec.rb ?

Reply:
Good catch!! 🤦🏻